src/command_modules/azure-cli-lab/azure/cli/command_modules/lab/custom.py

```diff
@@ -0,0 +1,167 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+import getpass
+from azure.cli.core._util import CLIError
+import azure.cli.core.azlogging as azlogging
+from azure.graphrbac import GraphRbacManagementClient
+
+logger = azlogging.get_az_logger(__name__)
+
+
+# pylint: disable=too-many-locals, unused-argument, too-many-statements
+def create_lab_vm(client, resource_group, lab_name, name, notes=None, image=None, size=None,
+                  admin_username=getpass.getuser(), admin_password=None,
+                  ssh_key=None, authentication_type='password',
+                  vnet_name=None, subnet=None, disallow_public_ip_address=None, artifacts=None,
+                  location=None, tags=None, custom_image_id=None,
+                  is_authentication_with_ssh_key=False, lab_virtual_network_id=None,
+                  os_publisher=None, os_offer=None, os_sku=None, os_version=None, os_type=None,
+                  gallery_image_reference=None, generate_ssh_keys=None):
+    """
+    :param client: Virtual machine client
```

Comment:
Why are you placing the help text here?  This results in duplication that is difficult to maintain.